server/services/store/sqlstore/data_migrations.go

```diff
@@ -868,8 +869,17 @@ func (s *SQLStore) doesDuplicateCategoryBoardsExist() (bool, error) {
 }
 
 func (s *SQLStore) runMySQLDeDuplicateCategoryBoardsMigration() error {
-	query := "DELETE FROM " + s.tablePrefix + "category_boards WHERE id NOT IN " +
-		"(SELECT * FROM ( SELECT min(id) FROM " + s.tablePrefix + "category_boards GROUP BY user_id, board_id ) as data)"
+	validatedTablePrefix := s.tablePrefix
```

Comment:
nit: doesn't seem there was any validation done on this, could we remove that in the var name so it's not misleading?
```suggestion
	tablePrefix := s.tablePrefix
```

Reply:
Done.

Reply:
Are you missing a push of the changes? I still see the same

Reply:
It's already there, it's a part of this commit https://github.com/mattermost/focalboard/pull/5009/commits/ae9413a2421036f4ce6e6b96b473e27802fe4d2d

Reply:
I see it now, it was a caching issue with Github